### DISCUSSION TITLE
按键QWE的投币点赞收藏效果怎么屏蔽？

### DISCUSSION THREAD

**Discussion by icateav** [the1812/Bilibili-Evolved#4947] [提问 / Q&A] [answered]
快捷键扩展好像没法屏蔽这几个按键的效果，是我漏掉了哪些设置吗

**Comment by the1812** [comment]
没有这种功能, 这和快捷键扩展没有关系

**Reply by Geeyun-JY3** [reply]
[`快捷键扩展 - 无动作`](https://github.com/the1812/Bilibili-Evolved/blob/77ada2305f3a4fba1c63dbe19cf08914cc787f38/doc/features/features.md#快捷键扩展---无动作)不就是干这个的吗？

**Reply by the1812** [reply]
想成一键三连的效果了... 如果只是屏蔽单个按键的话, 装无动作就行

**Reply by icateav** [reply]
我确实这样设置了 但是无意按倒QWE还是会触发点赞投币收藏的效果
![屏幕截图 2024-10-06 150320](https://github.com/user-attachments/assets/3ee4f0be-8cc0-408b-8ce6-312f53017224)

**Reply by the1812** [reply, marked as answer]
你要装无动作绑到 QWE 上，上面只是禁用脚本的快捷键
